packages/components/text/src/Variants.ios.ts

```diff
@@ -1,43 +1,57 @@
 import { Text } from './Text';
 
+// TODO(#2268): Remove "as any" designations once RN Core properly supports Dynamic Type scaling
+
 export const Caption1 = Text.customize({
   variant: 'caption1',
-});
+  dynamicTypeRamp: 'footnote',
```

Comment:
We should probably have a way to reconcile this in the future (i.e: `variant` sets dynamicTypeRamp`). I don't know if that's possible in the current PR

Reply:
If we wanted to do this in appleTypography.ios.ts, we would need to wait until `dynamicTypeRamp` becomes part of React Native's `TextStyle` (which should happen in 0.71), so this isn't possible right now.